update_ul17based.py

```diff
@@ -51,6 +76,26 @@
                 update_response = mcm.update('requests', request_ul18)
                 logger.info('Update response (UL18): %s', update_response)
 
+                present_ul18 = True
+
+        if  not present_ul18:
+
+            chained_request_id = request_ul17['member_of_chain'][0]
+            chained_request = mcm.get('chained_requests', query='prepid=%s' %chained_request_id)
+            root_id = chained_request[0]['chain'][0]
+
+            cursor.execute('INSERT INTO missing_ul VALUES (?, ?, ?, ?, ?, ?, ?)',
```

Comment:
This script is inserting values into the table every time it runs. Doesn't this duplicate data in the database?